src/kibana/plugins/visualize/editor/sidebar.html

```diff
@@ -1,7 +1,8 @@
 <div class="sidebar-container">
   <form class="sidebar-list"
         ng-submit="visualizeEditor.$invalid ? dontApply() : stageEditableVis()"
-        name="visualizeEditor">
+        name="visualizeEditor"
+        novalidate><!-- see http://goo.gl/9kgz5w -->
```

Comment:
When a form requires validation and has errors upon submission, Chrome tries to focus on the field that is causing errors. In some cases, the error is caused by a field that is `type="hidden"`, so it's not focusable. Adding `novalidate` removes this functionality (which only Chrome seems to support anyway), but doesn't seem to negatively impact our form validation. I've done some testing around this, but I probably didn't try everything. It is probably best to go through each agg and briefly test form validation and make sure it still functions correctly.